src/components/Persona/Persona.scss

```diff
@@ -169,30 +184,55 @@ $ms-Persona-presenceSizeXl: 28px;
 .ms-Persona-presence {
   background-color: $ms-color-presence-available;
   position: absolute;
-  height: 12px;
-  width: 12px;
+  height: $ms-Persona-presenceSizeMd;
+  width: $ms-Persona-presenceSizeMd;
   border-radius: 50%;
   top: auto;
   left: 34px;
```

Comment:
I know this isn't necessarily in the scope of your PR--but making a note that this needs to be RTL-ified as well.